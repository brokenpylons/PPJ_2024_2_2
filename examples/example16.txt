var n = 0;
for (var i = 1, 5) {
    print(n);
    n = n + 2;
}
