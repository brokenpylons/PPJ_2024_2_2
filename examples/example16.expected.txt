real("2") pow("^") real("3") 
